src/css/application.css

```diff
@@ -0,0 +1,433 @@
+.underline-span {
+  position: relative;
+  display: inline-block;
+}
+
+.underline-span::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0px;
+  height: 35%;
+  width: 100%;
+  background-color: var(--c-light-green);
+  transition: background-color 0.3s;
+  z-index: -1;
+  border-radius: 30px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
+}
+
+/* .container {
+  max-width: 375px;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+@media only screen and (min-width: 767.98px) {
+  .container {
+    max-width: 768px;
+    padding: 0 32px;
+  }
+}
+@media only screen and (min-width: 1199.98px) {
+  .container {
+    max-width: 1280px;
+    padding: 0 64px;
+  }
+} */
+
+/* .section {
+  padding: 40px 0;
+}
+@media only screen and (min-width: 767.98px) {
+  .section {
+    padding: 50px 0;
+  }
+}
+@media only screen and (min-width: 1199.98px) {
+  .section {
+    padding: 75px 0;
+  }
+}
+
+@media only screen and (min-width: 1199.98px) {
+  .feedback > .container {
+    padding: 0 128px;
+  }
+} */
+
+.feedback_form_container {
+  display: flex;
+  gap: 32px;
+  justify-content: center;
+}
+@media only screen and (max-width: 1199.98px) {
+  .feedback_form_container {
+    /* display: block; */
+  }
+}
+
+.feedback_advertising {
+  position: relative;
+
+  border-radius: 30px;
+  background: #4dc274;
+  overflow: hidden;
+
+padding: 20px 40px;
+
+  width: 492px;
+
+  flex-shrink: 0;
+}
+@media only screen and (max-width: 1199.98px) {
+  .feedback_advertising {
+    display: none;
+  }
+}
+
+.feedback_advertising_image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;  
+  height: auto;  
+
+}
+
+.kek {
+  margin-bottom: auto;
+  
+}
+
+.feedback_advertising_header {
+  font-family: Manrope;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 1.2;
+  letter-spacing: -0.02em;
+  text-align: center;
+
+  color: #ffffff;
```

Comment:
pls use var(--c-white) 
and use var() for al other colors (all defined in base.css)